### DISCUSSION TITLE
Is there a way to change the 30 day renewal threshold?

### DISCUSSION THREAD

**Discussion by johnrhunt** [adferrand/dnsrobocert#1184] [Q&A]
We're using dnsrobocert in an azure environment - storing the generated certs in a keyvault. There's a cool alerts feature that can SMS us when a cert has less than 30 days validity remaining..

Annoyingly for the certs to renew they have to get down below 30 days with dnsrobocert.

Is there a way to change this threshold to say, 31 days? I can do it via force renewal, but it's not very elegant. I couldn't see anything re this in the docs (or for certbot).

Loving dnsrobocert! Thanks

**Comment by OdinVex** [comment]
I think this?:

`acme:`
  `crontab_renew: ...`
